install/install.sh

```diff
@@ -1,157 +1,119 @@
-elif [ -x "$(command -v yum)" ]; then
-    error "Warning: Support for RHEL-like distros is experimental and things might break. Giving you 10 seconds to change your mind (by pressing Ctrl+C)..."
-    sleep 10
-
-    PACKAGE_MANAGER="yum"
-elif [ -x "$(command -v pacman)" ]; then
-    info "redpwn uses arch too btw"
-
-    PACKAGE_MANAGER="pacman"
-else
-    # XXX: should we stop the script here?
-    error "Unable to identify usable package manager, ignoring dependencies for now.";
-fi
-
-
-# setup variables
+  fi
 
-
-info "Configuring installation..."
-
-INSTALL_PATH="${INSTALL_PATH:-"/opt/rctf"}"
-
-if [ ! -d "$(dirname "$INSTALL_PATH")" ]; then
-    error "The parent of \$INSTALL_PATH ($(dirname "$INSTALL_PATH")) does not exist."
+  if [ ! -x "$(command -v curl)" ]; then
+    error "curl is not available. You must have curl to install rCTF."
     exit 1
-fi
-
-if [ -d "$INSTALL_PATH" ]; then
-    error "rCTF appears to already be installed in ${INSTALL_PATH}"
+  fi
 
-    info "... If you're trying to start rCTF, run 'docker-compose up -d --project-directory $INSTALL_PATH'."
-    info "... If you're trying to reinstall rCTF, 'rm -rf $INSTALL_PATH' then re-run this script."
+  RCTF_INSTALL_PATH="${RCTF_INSTALL_PATH:-"/opt/rctf"}"
 
+  if [ ! -d "$(dirname "$RCTF_INSTALL_PATH")" ]; then
+    error "The parent of \$RCTF_INSTALL_PATH ($(dirname "$RCTF_INSTALL_PATH")) does not exist."
     exit 1
-fi
-
-REPOSITORY_URL="${REPOSITORY_URL:-"https://github.com/redpwn/rCTF.git"}"
-REPOSITORY_BRANCH="${REPOSITORY_BRANCH:-"master"}"
-
-
-# install dependencies
-
+  fi
 
-info "Installing dependencies..."
+  if [ -d "$RCTF_INSTALL_PATH" ]; then
+    error "rCTF appears to already be installed in ${RCTF_INSTALL_PATH}"
 
-if [ "$PACKAGE_MANAGER" = "apt-get" ]; then
-    apt-get update
-    apt-get install --yes docker.io docker-compose git
-elif [ "$PACKAGE_MANAGER" = "yum" ]; then
-    info "We are about to install docker via https://get.docker.com/. Please follow along the steps to ensure it is configured properly."
-
-    # pass Ctrl+C / SIGINT to inside script
-    sh -c '
-        trap break INT
-        curl -fsSL https://get.docker.com/ | sh
-    '
-
-    yum install git
-elif [ "$PACKAGE_MANAGER" = "pacman" ]; then
-    pacman -Sy --noconfirm --needed docker docker-compose git
-fi
-
-info "Enabling docker..."
-
-(systemctl enable docker || true) 2>/dev/null # XXX: Debian "masks" docker.service
-(systemctl start docker || true) 2>/dev/null
+    info "... If you're trying to start rCTF, run 'docker-compose up -d'."
+    info "... If you're trying to reinstall rCTF, 'rm -rf $RCTF_INSTALL_PATH' then re-run this script."
 
+    exit 1
+  fi
 
-# clone repository
+  mkdir "$RCTF_INSTALL_PATH"
+  cd "$RCTF_INSTALL_PATH"
 
+  info "Installing dependencies..."
 
-info "Cloning repository to ${INSTALL_PATH} from repository ${REPOSITORY_URL} branch ${REPOSITORY_BRANCH}..."
+  if [ ! -x "$(command -v docker)" ]; then
+    curl -fsS https://get.docker.com | sh
```

Comment:
We may wish to prompt first and give users an option to bail and install themselves